include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -63,23 +63,23 @@ extern "C" {
 // Copied from TensorProto::DataType
 // Currently, Ort doesn't support complex64, complex128, bfloat16 types
 typedef enum ONNXTensorElementDataType {
-  ONNX_TENSOR_ELEMENT_DATA_TYPE_UNDEFINED = 0,
```

Comment:
I think keeping the numbers here, would be easier to make ABI backward compatible. For example, if someone inserted an item in the middle, the numbers wouldn't get shifted.

Reply:
Nobody should be using the numbers directly right? Yuan and I thought that the numbers were redundant and you have to think about the numbers if you ever edit it (since why are they specified if they are all exactly what the enum would use anyways?)